en/deploy/README.md

```diff
@@ -2,14 +2,24 @@
 
 > **Note** The following chapter can be sometimes a bit hard to get through. Persist and finish it; deployment is an important part of the website development process. This chapter is placed in the middle of the tutorial so that your mentor can help with the slightly trickier process of getting your website online. This means you can still finish the tutorial on your own if you run out of time.
 
-Until now your website was only available on your computer, now you will learn how to deploy it! Deploying is the process of publishing your application on the Internet so people can finally go and see your app :).
+Until now, your website was only available on your computer.  Now you will learn how to deploy it! Deploying is the process of publishing your application on the Internet so people can finally go and see your app :).
 
 As you learned, a website has to be located on a server. There are a lot of server providers available on the internet. We will use one that has a relatively simple deployment process: [PythonAnywhere](http://pythonanywhere.com/). PythonAnywhere is free for small applications that don't have too many visitors so it'll definitely be enough for you now.
 
 The other external service we'll be using is [GitHub](http://www.github.com), which is a code hosting service. There are others out there, but almost all programmers have a GitHub account these days, and now so will you!
 
-We'll use GitHub as a stepping stone to transport our code to and from PythonAnywhere.
+In the future, after completing this section, you will follow these steps to update your code on PythonAnywhere:
 
+1. Modify your program on your own computer
+2. Test your program (`python manage.py runserver`)
+3. Commit your chages to Git using `git add -A .` and `git commit -m "[Commit message]"`
+4. Push your changes to GitHub, using `git push` 
+5. At PythonAnywhere, open a bash console and change the working directory to your program (`cd djangogirls`)
+6. In the bash console, use Git to contact GitHub and get the latest version of your program with `git pull`
+7. Use the PythonAnywhere "Web" tab to reload your program
+8. Check the new, public version of your program for any problems
+
+Remember that the posts and users in the public version will be different than the posts and users you see on your own computer.
```

Comment:
I'm not a fan of this additional explanation, to be honest. I don't think it is adding any value, and it shows this process as complicated, long, and scary. 

I agree that we should add something about the fact that database on production is different than locally, so +1 on adding "Remember that the posts and users in the public version will be different than the posts and users you see on your own computer.", but not the rest of it.